jbpm-services/jbpm-services-ejb/jbpm-services-ejb-impl/src/main/java/org/jbpm/services/ejb/impl/RuntimeDataServiceEJBImpl.java

```diff
@@ -107,4 +111,10 @@ public void onDeactivate(DeploymentEvent event) {
 		super.onDeactivate(event);
 	}
 
+    @Override
+    public TaskSummaryQueryBuilder taskSummaryQuery( String userId ) {
```

Comment:
it's fine to have it here, it should not be exposed on Remote interface of this service, local ejbs should still allow to use query builders. The only thing I am wondering if they should be accessible via service interface or standalone factory or so....